mast/models.py

```diff
@@ -1 +1,69 @@
-# ADD DB CLASSES
\ No newline at end of file
+from mast import db, login_manager
+from flask_login import UserMixin
+import enum
+
+
+@login_manager.user_loader
+def load_user(user_id):
+    return db.session.query(User).get(int(user_id))
```

Comment:
Is explicit conversion, by calling `int()` initialization, really necessary?

Reply:
removed